components/google-cloud/google_cloud_components/mb_sdk_component/remote_runner.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for remote execution of Model Builder SDK as a pipeline component."""
+
+import argparse
+import inspect
+import os
+from google.cloud import aiplatform
+from google.cloud import storage
+
+INIT_KEY = 'init'
+METHOD_KEY = 'method'
+
+
+def split_args(kwargs):
+  """Split args to match MB SDK input format for init, method and config."""
+  init_args = {}
+  method_args = {}
+  config_args = {}
+
+  for key, arg in kwargs.items():
+    if key.startswith(INIT_KEY):
+      init_args[key.split('.')[-1]] = arg
+    elif key.startswith(METHOD_KEY):
+      method_args[key.split('.')[-1]] = arg
+    else:
+      config_args[key.split('.')[-1]] = arg
+
+  return init_args, method_args, config_args
+
+
+def write_to_gcs(project, gcs_uri, text):
```

Comment:
Just FYI https://github.com/kubeflow/pipelines/blob/eb3a092b64472b235e12eb96bcb0eda161b0b509/sdk/python/kfp/containers/_gcs_helper.py#L50 contains very similar logic. On the other hand it also makes sense to reduce the cross-dependencies. Up to you.

Reply:
Acknowledged, thank you.